.github/workflows/ci.yml

```diff
@@ -19,11 +19,6 @@ jobs:
           cd devstack
           sed -i 's/:cached//g' ./docker-compose-host.yml
           make dev.clone.https
-          cd ../edx-platform
-          git remote add mitodl https://github.com/mitodl/edx-platform.git
-          git fetch mitodl
-          git checkout master
-          cd ../devstack
```

Comment:
We used to run the tests on specific branches of our own fork, but that was because we had to cherry-pick some commits for the xBlock to work. Later on, those commits were [merged upstream](https://github.com/openedx/edx-platform/pull/19089/files) but we probably never changed the testing steps. This git-flow for testing was migrated from Travis.

I've changed it to run on upstream master now.

Reply:
Can you tell me which line of code in action here reflect this comment ?

> I've changed it to run on upstream master now. This also ensures that if the xBlock fails because of some changes upstream we will know it in the early stages.

Reply:
@asadiqbal08 https://github.com/mitodl/rapid-response-xblock/pull/126/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR21. We used to change remote to mitodl and then run tests, now we don't. So a `dev.clone.https` do this since we clone the master of devstack so the master of the `edx-platform` is fetched as well - just like edX works.